content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -25,7 +25,7 @@ This document describes the current state of `PersistentVolumes` in Kubernetes.
 
 ## Introduction
 
-Managing storage is a distinct problem from managing compute instances. The `PersistentVolume` subsystem provides an API for users and administrators that abstracts details of how storage is provided from how it is consumed. To do this, we introduce two new API resources:  `PersistentVolume` and `PersistentVolumeClaim`.
+Managing storage is a distinct problem from managing compute instances. The `PersistentVolume` subsystem provides an API for users and administrators that abstracts details of how storage is provided from how it is consumed. To do this, we introduce two API resources:  `PersistentVolume` and `PersistentVolumeClaim`.
```

Comment:
@oke-py , I am wondering if the introduction could be reworded. Typically, resource names are not emphasized in back ticks. What is the PersistentVolume subsystem? Is the PersistentVolume subsystem represented by the two resources?

Possible edit:
Managing storage is a distinct problem from managing compute instances. The Persistent volume APIs, PersistentVolume and PersistentVolumeClaim, abstract the details of how storage is provided from how storage is consumed.

A PersistentVolume (PV) is a ....

A PersistentVolumeClaim (PVC) ...

While PersistentVolumeClaim resources ...